src/Sulu/Bundle/TestBundle/Testing/TestUserProvider.php

```diff
@@ -87,14 +87,16 @@ public function getUser()
             $this->entityManager->persist($contact);
 
             $user = $this->userRepository->createNew();
-            $this->setCredentials($user);
-            $user->setLocale('en');
             $user->setContact($contact);
             $this->entityManager->persist($user);
-        } else {
-            $this->setCredentials($user);
         }
 
+        $user->setUsername(self::TEST_USER_USERNAME);
```

Comment:
no sure if you commited this on purpose?

Reply:
I did, this is the fix for [this error](https://travis-ci.org/github/sulu/sulu/jobs/694915779#L1757).

It especially is about the `setLocale` below. One of the tests of the SecurityBundle sets the locale to another language, and it will stay like that for all tests afterwards. For this reason the tests were failing if all tests are executed, but not if only the SnippetBundle tests were executed.